### DISCUSSION TITLE
Menu account not is complete

### DISCUSSION THREAD

**Discussion by Danieleeffe1** [OpenMage/magento-lts#2900] [Questions & Answers] [answered]
Good morning everyone

I have a magento 1.9.4.3 installation updated to openmage 19.4.21
ubuntu server 20.04
php 7.4
redis 6.6

i have a strange situation in the frontend
Description
1) the search field has disappeared. Io is using a custom theme based on RWD, but even if I switch to default RWD the search field doesn't appear.

2) on the home page, if a customer clicks on the ACCOUNT icon, he sees the complete menu list:

my account
my order
CheckOut
Sign in
Log in

if it's on any other page, it only sees the menus:

my order
CheckOut
Log in

if you log in the situation does not change (except the menu item "Login" becomes "Logout")

if I refresh the page it sometimes appears (not always) "my account"

i tried disabling all magento caches and external caches (redis, memcache, magento cache page external) and system caches, but the result doesn't change.

Has this ever happened to anyone? how did you solve it? How can I investigate?

PS:
I understand that for you the magento backup is not important, but it is not so for everyone. decomposing the Mage_All file into different files will also have its advantages but completely deleting the Back-Up module entry is incomprehensible.

[the remark this answers is not in the document]
**Reply by fballiano** [reply]
Mage backup is already in the release builder so there’s no difference for who downloads OM via zip file, via composer you just newd to require the module 😉

[the remark this answers is not in the document]
**Reply by fballiano** [reply]
The reason why it was separated from the main repo is in  #2259

[the remark this answers is not in the document]
**Reply by elidrissidev** [reply]
Release v19.4.21 is at least two weeks old, it doesn't have that change?! I'm not connecting the dots here.

[the remark this answers is not in the document]
**Reply by fballiano** [reply]
19.4.21 contained only https://github.com/OpenMage/magento-lts/issues/2793 and no other change.

[the remark this answers is not in the document]
**Reply by fballiano** [reply]
and in 19.4.20 mage_backup wasn't removed

[the remark this answers is not in the document]
**Reply by Danieleeffe1** [reply]
Thanks everyone for the replies, but the main question of the post is about the user menu and the search field

[the remark this answers is not in the document]
**Reply by addison74** [reply]
If he only uses the git command to update his local files then it's really a problem, because everything related to this module will be removed. Although I was a supporter of keeping this module I agreed to keep it separate and I don't regret the decision. The arguments I have got were those mentioned by @fballiano before, it exists in the periodically released versions and it can be installed with Composer. Also, this change has been announced, a developer should learn about all changes and he must do routine testing before updating a production version. Anyone who has an issue with the new versions will receive an answer as can be seen very quickly.

**Comment by addison74** [comment]
In such situations it is very difficult to quickly establish the source of the problem. I have versions of Magento and OM installed in a test environment (DDEV) each with the Magento sample pack. If I notice an issue, I can immediately identify which version it appeared on, then I check the list of PRs included in that version. After I created a shorter list, I revert one by one those commits. 

So far I have checked the following in Frontendm version 19.4.21:

1. The **search box** is displayed with the default RWD theme on all pages.

2. The **Account** dropdown menu has the same list of links on all pages.

These have nothing to do with modules recently removed from the repository.

**Reply by Danieleeffe1** [reply]
small step forward,

i restored my backup
then I copied and pasted the files from openmage 19.4.21 leaving out the files in the app/frontend/etc/modules folder

I renamed my Mage_All.xml file to Mage_all(without extension)

I copied all openmage 1.9.4.21 Mage_*.xml files and pasted them in "app/frontend/etc/modules" folder and the searc field and menu have the problems indicated in the post

**Reply by addison74** [reply]
Can you reproduce this issue if you install OpenMage 19.4.21 from scratch? Without anything else.

**Reply by Danieleeffe1** [reply]
ok....

I have identified the problem

Mage_Page.xml file is not loaded

the file has permissions are 644
I also tried with 777 permissions but nothing

I had to reset the Mage_All.xml file with just the Mage_Page module instructions and now the search field and user menu are normal.

mysteries

**Reply by Danieleeffe1** [reply]
I answer the question

"Have you tried a clean install?"

I tried a clean installation and the problem does not occur, but given the cause I don't think it depends on an installation of magento updated to openmage

**Reply by addison74** [reply]
All the changes related to this file are here https://github.com/OpenMage/magento-lts/commits/1.9.4.x/app/etc/modules/Mage_Page.xml. There is nothing wrong. It depends on the Mage_Core module, but I doubt it is disabled.

If by installing from scratch you did not reproduce the issue, then it means that you did something in the source code. You should start using Git to have a record of the changes that you operate locally.

**Reply by Danieleeffe1** [reply]
I'm not a developer and I never touch the source code, I can imagine it's an add-on.

it's strange that only restoring the Mage_All file with the Mage_Page code portion fixes the problem.

[the remark this answers is not in the document]
**Reply by sreichel** [reply, marked as answer]
@Flyingmana Mage_Backup is still present in all releases. Only removed in dev-main and v20 branch.
